### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_Operatortab_ScreenReader:Screen reader is announcing the column header twice for the table.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5665]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
GitHubTags: #A11yMAS; #A11yTCS; #MAS4.2.3; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp#DesktopApp
**Environment Details:**
Application name: Azure Data Studio (Agent, Create External Data, Notebooks)
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS : MacOS Mojave 10.14.5

**Additional Details:**
Screen reader name: Voice Over
MAS reference: MAS1.3.1, MAS4.2.3

**Repro Steps:**
1.Launch the Azure Data Studio application.
2.Enable the SQL server agent extension.
3.Navigate to the connected server->Select the database->Navigate to the SQL agent tab.
4. Navigate to the Operator tab->Use the next tab Navigate to table.
5. Observe that Screen reader is announcing the column header twice for the table.

**Actual:**
Screen reader is announcing the column header twice for the table.

**Similar issue is observed for the below pages:**
**1.Jobs:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->Navigate to jobs page.
**2.Proxiestab:** Navigate to the connected server->Select the database->Navigate to the SQL agent tab->Navigate to Proxies tab.
3.**CSVPreviewData**-Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button.
4.**CSVSummary** - Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV->Click on the "Next" button->click on next button.

**Expected:**
Screen reader should announce the column header once for the table.

**User Impact:**
User with disabilities will not be able to understand why the screen reader reading that column header twice for the table.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
**MAS4.2.3**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}

**MAS1.3.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D&cid=1a1d7dfd-3dfd-4394-a8af-81fa7c1cf482

**Attachment for Reference:**
[5665_[Accessibility]A11y_AzureDataStudio_Operatortab_ScreenReader..pptx](https://github.com/microsoft/azuredatastudio/files/3223854/5665_.Accessibility.A11y_AzureDataStudio_Operatortab_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes
